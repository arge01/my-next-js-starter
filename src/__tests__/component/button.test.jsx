import { render } from '@testing-library/react';

import Button from '@component/Button';

it('renders homepage unchanged', () => {
  const { container } = render(
    <Button onClick={() => 'button'}>This a button</Button>
  );
  expect(container).toMatchSnapshot();
});
