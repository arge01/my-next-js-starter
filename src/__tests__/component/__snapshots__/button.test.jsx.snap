// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <button>
    This a button
  </button>
</div>
`;
